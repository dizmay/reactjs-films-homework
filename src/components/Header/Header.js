import React from 'react';
import SearchInput from '../SearchInput/SearchInput';
import styles from './Header.scss';

const Header = () => (
  <header className={styles.header}>
    <h2 className={styles.header__logo}>FILMS</h2>
    <SearchInput />
  </header>
);

export default Header;
