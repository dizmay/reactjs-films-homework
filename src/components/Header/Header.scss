* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

.header {
  width: 100%;
  display: flex;
  justify-content: space-between;
  align-items: center;
  background: linear-gradient(180deg, rgba(0, 0, 0, 0.5) 30%, rgba(255, 255, 255, 0) 100%);
  padding: 1rem 10rem 0;
}

.header__logo {
  color: #fff;
  font-size: 2rem;
}
