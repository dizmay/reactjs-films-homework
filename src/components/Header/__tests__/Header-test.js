import React from 'react';
// import renderer from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';
import Header from '../Header';

it('renders correctly', () => {
  const renderer = new ShallowRenderer();
  const tree = renderer.render(<Header />);
  expect(tree).toMatchSnapshot();
});
