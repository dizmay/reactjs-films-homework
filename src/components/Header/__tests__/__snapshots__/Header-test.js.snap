// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="header"
>
  <h2
    className="header__logo"
  >
    FILMS
  </h2>
  <Memo(Connect(SearchInputContainer)) />
</header>
`;
